### PR TITLE
fix #62747 shiachat.com

### LINKED ISSUE #62747: shiachat.com
### Issue URL (Ads)
 [https://www.shiachat.com/forum/topic/234973261-rained-blood-wrong-year/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.shiachat.com%2Fforum%2Ftopic%2F234973261-rained-blood-wrong-year%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard for Windows v7.5.1
Browser: | Edge 85.0.564.41 (Chromium-based)
AdGuard driver: | WFP
Stealth mode options: | Hide your search queries, <br/>Send Do-Not-Track header, <br/>Remove X-Client-Data header from HTTP requests, <br/>Self-destructing third-party cookies (180), <br/>Block WebRTC, <br/>Block Push API, <br/>Block Location API, <br/>Hide your User-Agent ()
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Social Widgets:</b><br/> AdGuard Social Media<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> Liste AR
Other extensions used: | MS AntiVirus

### PR DESCRIPTION
**Pull request by stanislav-atr**
#62747

### CODE REVIEW

File: AnnoyancesFilter/sections/popups.txt [hunk 1]
```diff
@@ -182,6 +182,7 @@ mamasepeti.com#$#body { overflow: visible!important; }
 ibe.kr#$##ouibounce-modal { display: none!important; }
 ibe.kr#$#.modal-backdrop { display: none!important; }
 ibe.kr#$#body { overflow: visible!important; }
+shiachat.com##body > #ipsLayout_header > .cAnnouncementsFloat
```
Comment (zloyden) [inline AnnoyancesFilter/sections/popups.txt:185]: Just this is enough `shiachat.com###ipsLayout_header > div.cAnnouncementsFloat`

File: EnglishFilter/sections/specific.txt [hunk 2]
```diff
@@ -595,6 +595,7 @@ swordz.io##div[style="width:300; height: 250; background-color: #000"]
 disasterscans.com##div[id^="media_image-"].no-icon.heading-style-1
 9tsu.net##.ads-system
 pcworld.com###amazon-bottom-widget
+shiachat.com##tr > td > div[class^="ipsA"]
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:598]: Try to use more easy way to block it than you have. Something like this `shiachat.com##body > div#ipsLayout_header + center` and additionally block gif images via URL

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 2 above)
Comment (stanislav-atr) [reply EnglishFilter/sections/specific.txt]: I've modified the rule, but i am not sure about gifs. Many of them are userpics.

Comment (zloyden) [reply EnglishFilter/sections/specific.txt]: I don't mean all the gifs, just those ads.
For example, you can block these urls 
```
https://www.shiachat.com/forum/uploads/monthly_2020_07/1664101338_alaligemsa.gif.778f6887c1477add141fc4129dfded51.gif
https://www.shiachat.com/forum/uploads/monthly_2015_11/shiamatch.gif.a9bfbde53248764fbf95258566652208.gif
```

File: EnglishFilter/sections/specific.txt [hunk 3]
```diff
@@ -595,7 +595,7 @@ swordz.io##div[style="width:300; height: 250; background-color: #000"]
 disasterscans.com##div[id^="media_image-"].no-icon.heading-style-1
 9tsu.net##.ads-system
 pcworld.com###amazon-bottom-widget
-foes.io###adParent
+shiachat.com##body > div#ipsLayout_header + centerfoes.io###adParent
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:598]: Something weird here

**Review by zloyden — CHANGES_REQUESTED** [review]

File: EnglishFilter/sections/specific.txt (hunk 3 above)
Comment (stanislav-atr) [reply EnglishFilter/sections/specific.txt:598]: Fastest ctr+v in the wild wild west

**Comment by stanislav-atr** [comment]
Is that ok or too broad?

File: EnglishFilter/sections/specific.txt [hunk 4]
```diff
@@ -595,6 +595,8 @@ swordz.io##div[style="width:300; height: 250; background-color: #000"]
 disasterscans.com##div[id^="media_image-"].no-icon.heading-style-1
 9tsu.net##.ads-system
 pcworld.com###amazon-bottom-widget
+shiachat.com##body > div#ipsLayout_header + center
+||shiachat.com/forum/uploads/monthly_*/*.gif
```
Comment (zloyden) [inline EnglishFilter/sections/specific.txt:599]: It's kinda wide. Make 2 rules.
For example `||shiachat.com/forum/uploads/monthly_*/shiamatch.*.gif` or you can match them via Regex.

**Review by zloyden — CHANGES_REQUESTED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**